quickwit/quickwit-search/src/search_response_rest.rs

```diff
@@ -32,50 +33,52 @@ pub struct SearchResponseRest {
     /// Overall number of documents matching the query.
     pub num_hits: u64,
     /// List of hits returned.
-    pub hits: Vec<serde_json::Value>,
+    pub hits: Vec<Value>,
```

Comment:
`serde_json::Value` is better or use `JsonValue` (`use serde_json::Value as JsonValue`).